### ISSUE TITLE
docker.sh: line 1: 400:: command not found

### ISSUE THREAD

**Issue by Denniz** [AzuraCast/AzuraCast#1573]
**Using Docker installation method**
Yes

**Host Operating System**
Ubuntu 16.04

**Describe the bug**
Using the method described (https://www.azuracast.com/install.html#self-hosted-installation) I get an error when running `./docker.sh install`.
The error is `./docker.sh: line 1: 400:: command not found`.
Tried with and without `sudo`, also did `sudo -s` to login as 'sudo', error remains.

**To Reproduce**
`./docker.sh install`
or
`sudo ./docker.sh install`

**Expected behavior**
Install AzuraCast

**Additional context**
Also installed `docker` and `docker-compose` from the Ubuntu library, issue remains.
(Ubuntu 16.04 fully updates)

**Comment by SlvrEagle23** [comment]
@Denniz If you type:

```
cat docker.sh
```

...do you see a regular-looking file or an error message?

It's possible that you encountered an error while downloading the `docker.sh` utility script, in which case the easy resolution is to just try to download it again using the same installation steps as before.

**Comment by Denniz** [comment]
> @Denniz If you type:
> 
> ```
> cat docker.sh
> ```
> 
> ...do you see a regular-looking file or an error message?
> 
> It's possible that you encountered an error while downloading the `docker.sh` utility script, in which case the easy resolution is to just try to download it again using the same installation steps as before.

With `cat docker.sh` I get `400: Invalid request`, so I guess that means the file didn't download properly.
When I try `sudo curl -L https://raw.githubusercontent.com/AzuraCast/master/docker.sh > docker.sh` I get `-bash: docker.sh: Permission denied`.
I'm trying to download into `/var/azuracast/`
both `var` and `azuracast` folders are owned by `root:root` and have 755 permissions.

Because I got `-bash: docker.sh: Permission denied` I used `sudo -s` to up my permissions and download as root, which didn't give the `-bash` error but apparently didn't download the file correctly.

Any tips?
